ansible/all-in-one/roles/xsnippet/files/nginx.conf

```diff
@@ -0,0 +1,32 @@
+load_module modules/ngx_http_js_module.so;
+
+user  nginx;
+worker_processes  auto;
```

Comment:
It will create as many nginx workers as we have CPUs on the host. Should we probably have 1 or 2 workers all the time?

Reply:
Correct. It's one per CPU. This is the default config. The only reason I'm adding it here is that I need to add a single line (`load_module modules/ngx_http_js_module.so;`) above, and there is no other mechanism in the official nginx docker image to do that :(

Worker memory usage is negligible (~3 MiB of RSS), but yeah, we could lower it.